lib/fprotect/Kconfig

```diff
@@ -10,3 +10,11 @@ config FPROTECT
 	help
 	  Use hardware peripherals (BPROT, ACL, or SPU) to write-protect
 	  regions of flash until next reset.
+
+if FPROTECT
+config FPROTECT_BLOCK_SIZE
+	int "Block size of fprotect"
+	default 32768 if SOC_NRF9160
```

Comment:
Add a default for nRF53APP as well.

Reply:
53APP work with the DT define, 53NET however, needs a specific one, added that.

Reply:
ah forgot, it has SPU, nvm. Still need 53net xD